hack/release-ga.sh

```diff
@@ -72,6 +72,9 @@ do
 	EOF
 done
 
+# Bump minimal target version to GA one
+sed -i -e "s|z_min: .*|z_min: ${MAJOR}.${MINOR}.0|" "build-suggestions/${MAJOR}.${MINOR}.yaml"
```

Comment:
The GA version might not be .0, like 4.10.3:

```console
$ for Y in $(seq 8 13); do grep " 4[.]${Y}[.]" "channels/stable-4.${Y}.yaml" | sort -V | head -n1; done
- 4.8.2
- 4.9.0
- 4.10.3
- 4.11.0
- 4.12.0
- 4.13.0
```

We can probably drop the `exec`  below, run the stabilization script three times or some such, and then take the smallest version out of the new `stable-4.y.yaml` and use that in this injection?  Or maybe that's over-engineering now that we more frequently do GA 4.y.0?  Or maybe even when we don't GA 4.y.0, it's close enough to  be a min version?  Lot of options, and I don't feel strongly about the various trade-offs.

Reply:
haha I only looked back until 4.11

let me check whether I can easily do something smarter like you suggest. I avoided doing something smarter to avoid dependency on python yaml package, but I havent noticed that we do run stabilization-changes which already use yaml so I can as well use that